### ISSUE TITLE
5.1.1 installation fails

### ISSUE THREAD

**Issue by magikmw** [PrestonKnopp/language-gdscript#34]
5.1.1 installation fails from apm and settings gui with error below.

Atom 1.46.0 on Fedora 31
Haven't had this problem before, previously upgraded with no problems to 5.1.0

```
> tree-sitter-gdscript@1.0.0 install /tmp/apm-install-dir-120417-518189-dq6fl3.ja407/node_modules/lang-gdscript/node_modules/tree-sitter-gdscript
> prebuild-install || node-gyp rebuild


prebuild-install WARN install No prebuilt binaries found (target=4.2.7 runtime=electron arch=x64 libc= platform=linux)
gyp ERR! configure error 
gyp ERR! stack Error: Python executable "/usr/share/atom/resources/app/apm/bin/python-interceptor.sh" is v3.7.7, which is not supported by gyp.
gyp ERR! stack You can pass the --python switch to point to Python >= v2.5.0 & < 3.0.0.
gyp ERR! stack     at PythonFinder.failPythonVersion (/usr/share/atom/resources/app/apm/node_modules/npm/node_modules/node-gyp/lib/configure.js:501:19)
gyp ERR! stack     at PythonFinder.<anonymous> (/usr/share/atom/resources/app/apm/node_modules/npm/node_modules/node-gyp/lib/configure.js:483:14)
gyp ERR! stack     at ChildProcess.exithandler (child_process.js:282:7)
gyp ERR! stack     at ChildProcess.emit (events.js:182:13)
gyp ERR! stack     at maybeClose (internal/child_process.js:961:16)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:248:5)
gyp ERR! System Linux 5.6.8-200.fc31.x86_64
gyp ERR! command "/usr/share/atom/resources/app/apm/bin/node" "/usr/share/atom/resources/app/apm/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /tmp/apm-install-dir-120417-518189-dq6fl3.ja407/node_modules/lang-gdscript/node_modules/tree-sitter-gdscript
gyp ERR! node -v v10.2.1
gyp ERR! node-gyp -v v3.7.0
gyp ERR! not ok 
npm WARN enoent ENOENT: no such file or directory, open '/tmp/apm-install-dir-120417-518189-dq6fl3.ja407/package.json'
npm WARN ws@7.3.0 requires a peer of bufferutil@^4.0.1 but none is installed. You must install peer dependencies yourself.
npm WARN ws@7.3.0 requires a peer of utf-8-validate@^5.0.2 but none is installed. You must install peer dependencies yourself.
npm WARN apm-install-dir-120417-518189-dq6fl3.ja407 No description
npm WARN apm-install-dir-120417-518189-dq6fl3.ja407 No repository field.
npm WARN apm-install-dir-120417-518189-dq6fl3.ja407 No README data
npm WARN apm-install-dir-120417-518189-dq6fl3.ja407 No license field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! tree-sitter-gdscript@1.0.0 install: `prebuild-install || node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the tree-sitter-gdscript@1.0.0 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/magikmw/.atom/.apm/_logs/2020-05-17T21_10_40_290Z-debug.log```

**Comment by PrestonKnopp** [comment]
Hi, thanks for posting an issue. That's strange that it worked before but not now. This looks to be an atom / apm issue having trouble finding the right python.

I tried installing on Ubuntu, where I have `python2` and `python3` but no `python` and I received error not finding python. Symlinking `python` to `python3` produces the same error as in your post. 

The issue is fixed by symlinking `python` to `python2`.

---

I just uploaded binaries for linux. Try reinstalling normally and see if that works for you.
